ports/satlovecake/SATAN LOVES CAKE.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Set XDG_DATA_HOME if not already set
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+# Check for PortMaster installation in various directories
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+# PortMaster info
+source $controlfolder/control.txt
+export PORT_32BIT="Y"
```

Comment:
Is this a 32 bit gmloader next? If yes ignore this comment. Else line need to be removed.

Reply:
yes it's armhf build :)